### ISSUE TITLE
error when trying to run energy plus analysis

### ISSUE THREAD

**Issue by Sinasta** [rgsouthall/vi-suite07#6]
I am getting this error message when clicking on 'calculate' on the envi simulation node:
Python: Traceback (most recent call last):
  File "/home/sinasta/.config/blender/3.4/scripts/addons/vi-suite07-master/vi_operators.py", line 1856, in modal
    self.esimruns.append(Popen(self.esimcmds[self.e].split(), stderr=PIPE))
  File "/usr/lib/python3.10/subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.10/subprocess.py", line 1845, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
OSError: [Errno 8] Exec format error: 'energyplus'

Linux Arch, Blender 3.4, Python 3.10
![Screenshot from 2022-09-09 13-42-53](https://user-images.githubusercontent.com/64699866/189342800-33fc02b3-6a30-4c36-bdf8-834b62a27bcd.png)

**Comment by rgsouthall** [comment]
Have you downloaded from master very recently? I made some changes a few days ago to correct permissions settings for energyplus. 
Check the contents of ~/.config/blender/3.4/scripts/addons/vi-suite07-master/EPFiles/linux. You should have a energyplus-22.1.0 file which is executable and an energyplus file which is a symbolic link to it.
Bear in mind I have not tested VI-Suite with 3.4 yet and there maybe api changes so I am not accepting general bug reports with 3.4 yet.

**Comment by Sinasta** [comment]
Thank you for your answer. I did try the same with blender 3.3 but the error is the same. i checked the content of EPFiles/linux folder and everything is exactly as it should be. I am using the most recent version with your latest changements. I tried to use a version before this [commit](https://github.com/rgsouthall/vi-suite07/commit/2f8d27047a58b3c6066b95a9db71ec6087563178) but the error remains the same. I did verify if the energyplus-22.1.0 file is executable and if the 'energyplus' file is properly linking to it but everything is correct. 

![Screenshot from 2022-09-09 18-04-05](https://user-images.githubusercontent.com/64699866/189393530-f727e793-e28f-41c5-bb9d-51d9185e2f61.png)

![Screenshot from 2022-09-09 18-05-20](https://user-images.githubusercontent.com/64699866/189393564-f31639a0-9ad3-43b3-8104-1f08af26c479.png)
